internal/commands/pull/command_test.go

```diff
@@ -197,15 +200,30 @@ Successfully pulled app down: app
 		})
 
 		t.Run("should return the error when exporting dependencies fails", func(t *testing.T) {
-			profile, teardown := mock.NewProfileFromTmpDir(t, "pull_handler_test")
-			defer teardown()
+			t.Run(" when include package json file", func(t *testing.T) {
```

Comment:
[nit] unnecessary leading space here

Reply:
huh this is a github UI thing, I did not configure my vim to convert tab to space, so it looks fine in my editor but github treats a tab as 8 space.

Reply:
and the tabs are added by `go fmt` ...

Reply:
I'm referring to the `" "` in the `t.Run` description...
```suggestion
			t.Run("when include package json file", func(t *testing.T) {
```

Reply:
my stupidity is gonna drag me to hell one day...

Reply:
I'll appreciate the company 🙃